event-sourcing/event-subscription-registry/src/main/java/uk/gov/justice/subscription/registry/EventSourceDefinitionRegistry.java

```diff
@@ -1,39 +1,60 @@
 package uk.gov.justice.subscription.registry;
 
-import static java.util.Optional.ofNullable;
-import static java.util.stream.Collectors.toMap;
-
 import uk.gov.justice.subscription.domain.eventsource.EventSourceDefinition;
 
+import java.util.HashMap;
 import java.util.Map;
-import java.util.Optional;
-import java.util.function.BinaryOperator;
-import java.util.stream.Stream;
+
+import javax.enterprise.inject.Vetoed;
 
 /**
  * Registry containing {@link EventSourceDefinition}s mapped by the event source name
  */
+@Vetoed
```

Comment:
Does this class need to be Vetoed as it is not container managed

Reply:
@Vetoed removed